feat(changelog): private-networks-changed-virtual-private-cloud-is 2023-07-18 (#1939)
scaleway/docs-content: changelog/july2023/2023-07-18-private-networks-changed-virtual-private-cloud-is.mdx

---
title: Virtual Private Cloud is now in General Availability
status: changed
author:
    fullname: 'Join the #virtual-private-cloud channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2023-07-18
category: network
product: private-networks
---

> VPC allows you to quickly and seamlessly secure all your application architecture's resources of from external networks. Additionally, your infrastructure will benefit from improved resiliency as VPC is now regionalized across the Availability Zones by default.

Reviewer: "resources of from" -> "resources from"

> VPC allows you to quickly and seamlessly secure all your application architecture's resources of from external networks. Additionally, your infrastructure will benefit from improved resiliency as VPC is now regionalized across the Availability Zones by default.

Reviewer 2: ```suggestion
VPC allows you to quickly and seamlessly secure all your application architecture's resources from external networks. Additionally, your infrastructure will benefit from improved resiliency as VPC is now regionalized across the Availability Zones by default.
```

> Formerly named Private Networks, VPC allows you to secure quickly and seamlessly all the resources of your application's architecture from external networks. Additionally, your infrastructure will benefit from improved resiliency as VPC is now regionalized across the AZs by default.

Reviewer 2: ```suggestion
VPC allows you to quickly and seamlessly secure all your application architecture's resources of from external networks. Additionally, your infrastructure will benefit from improved resiliency as VPC is now regionalized across the Availability Zones by default.
```

> - By default, each region will have at least one VPC. You can create more if needed.

Reviewer 3: ```suggestion
- By default, each region will have at least one VPC. You can create more if needed
```

> - By default, each region will have at least one VPC, and you can create more if needed

Reviewer 2: ```suggestion
- By default, each region will have at least one VPC. You can create more if needed.
```

> - Each VPC can have multiple Private Networks, each with its own distinct CIDR blocks

Reviewer: missing full stop at the end of the sentence ?

> - VPC will now include a DHCP server for each Private Network

Reviewer: Inconsistent end punctuation
